mypy_django_plugin/lib/helpers.py

```diff
@@ -385,3 +389,34 @@ def add_new_manager_base(api: SemanticAnalyzerPluginInterface, fullname: str) ->
     if sym is not None and isinstance(sym.node, TypeInfo):
         bases = get_django_metadata_bases(sym.node, "manager_bases")
         bases[fullname] = 1
+
+
+def is_abstract_model(model: TypeInfo, /) -> bool:
```

Comment:
```suggestion
def is_abstract_model(model: TypeInfo) -> bool:
```

No reason to make it pos-only